Fix avatar display to not overwrite display style

juliusknorr opened the pull request:
We always show the avatar container, so we don't need the show() call. The image is added after successful loading. This was caused by #7498 since we reduced the two code paths to one.

Fix for https://github.com/nextcloud/spreed/issues/539

juliusknorr committed:
Remove show() function since we always show the avatar container

Signed-off-by: Julius Härtl <jus@bitgrid.net>

Ivansss reviewed (approved):
Tested and fixes the issue https://github.com/nextcloud/spreed/issues/539

rullzer commented:
I think this was added by @skjnldsv to fix showing a spinner somewhere if I remember correctly...

juliushaertl commented:
The loading spinner works everywhere, as far as I tested. But maybe @skjnldsv can have a look as well.

skjnldsv commented:
I think it was to force the avatar to have a display style property. I didn't test this pr though. I'm not sure. Please test with the avatar on the settings page. This were where it was tested when I added this. :)

juliushaertl commented:
> I think it was to force the avatar to have a display style property. I didn't test this pr though. I'm not sure. Please test with the avatar on the settings page. This were where it was tested when I added this. :)

I see no issues there. :wink:

skjnldsv commented:
Then it's perfect! 😍

nickvergessen commented:
Also fixes the activity stream 👍

nickvergessen commented:
@MorrisJobke should merge before final, otherwise activities look awkward:
> ![bildschirmfoto von 2018-01-08 10-21-01](https://user-images.githubusercontent.com/213943/34664824-fd9bc63c-f45d-11e7-94bf-7bc02a90feb9.png)

MorrisJobke commented:
Please fix the JSUnit tests before:

```
PhantomJS 2.1.1 (Linux 0.0.0) jquery.avatar tests valid avatar unhide div FAILED
	Expected 'none' to equal 'block'.
	core/js/tests/specs/jquery.avatarSpec.js:206:39
```

juliusknorr committed:
Fix tests

The avatar plugin should not change the display element, since the
avatar is always shown by default and the display value is up to the
developers

Signed-off-by: Julius Härtl <jus@bitgrid.net>
